back/engines/commercial/bulk_import_ideas/app/views/bulk_import_ideas/web_api/v1/export_form.html.erb

```diff
@@ -186,6 +202,26 @@
               <% end %>
             </div>
 
+          <% elsif field[:format] == :matrix_linear_scale %>
+            <div class="matrix">
+              <table>
+                <tr>
+                  <th></th>
```

Comment:
To be removed?

Reply:
No, the empty cell is needed above the statements